.github/workflows/examples-darwin.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build Example - Darwin
```

Comment:
The Darwin one says `Darwin`, the other examples follow the `Build Example - XXXXX` scheme. Happy to rename if you think its unclear here.

Reply:
I think my main concern is that this will be intepreted as "this is an example of how you build a Matter thing on Darwin", whereas that's not what this actually is, and most Matter things on Darwin are not built the way examples/tv-casting-app is.

I would suggest that we make it explicit in the naming (both of the workflow file and the workflow and the job) that this is pretty specific to tv-casting-app.

Reply:
Or perhaps something to make it clear that this is an example of how you could do things using the Matter SDK but not the Darwin Matter.framework, or something along those lines.